core/src/main/resources/jenkins/model/CoreEnvironmentContributor/buildEnv.properties

```diff
@@ -3,6 +3,7 @@ BUILD_ID.blurb=The current build ID, identical to BUILD_NUMBER for builds create
 BUILD_DISPLAY_NAME.blurb=The display name of the current build, which is something like "#153" by default.
 JOB_NAME.blurb=Name of the project of this build, such as "foo" or "foo/bar".
 JOB_BASE_NAME.blurb=Short Name of the project of this build stripping off folder paths, such as "foo" for "bar/foo".
+JOB_FOLDER_NAME.blurb=Full name of the parent folder of this build, such as "bar" for "bar/foo".
```

Comment:
Should explicitly mention what the folder name is in case of a top level job, and a three level nested hierarchy.

Reply:
Sure: done.